From 2b1a5a4937e6de3aeba31fdd96946ff66d3c1608 Mon Sep 17 00:00:00 2001
From: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
Date: Fri, 1 May 2020 23:23:49 +0200
Subject: [PATCH 19/96] dt-bindings: clock: meson8b: Add the Meson8 Ethernet
 (RMII) clocks

Export CLKID_ETH_CLK (and it's parents) because it is used as input for
the Ethernet controller on Meson8 SoCs.

Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
---
 include/dt-bindings/clock/meson8b-clkc.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/dt-bindings/clock/meson8b-clkc.h b/include/dt-bindings/clock/meson8b-clkc.h
index 385bf243c..a09b686af 100644
--- a/include/dt-bindings/clock/meson8b-clkc.h
+++ b/include/dt-bindings/clock/meson8b-clkc.h
@@ -221,5 +221,9 @@
 #define CLKID_VCLK2_EN		215
 #define CLKID_VID_PLL_LVDS_EN	216
 #define CLKID_HDMI_PLL_DCO_IN   217
+#define CLKID_ETH_CLK_SEL	218
+#define CLKID_ETH_CLK_DIV	219
+#define CLKID_ETH_CLK_PHASE	220
+#define CLKID_ETH_CLK		221
 
 #endif /* __MESON8B_CLKC_H */
-- 
2.45.1

